internal/bufferedread/buffered_reader.go

```diff
@@ -285,6 +285,7 @@ func (p *BufferedReader) ReadAt(ctx context.Context, inputBuf []byte, off int64)
 	defer func() {
 		dur := time.Since(start)
 		p.metricHandle.BufferedReadReadLatency(ctx, dur)
+		p.metricHandle.GcsReadBytesCount(int64(bytesRead), "Buffered")
```

Comment:
Could you sync the latest master and use the generated constants instead of free strings here and elsewhere except tests.